lib/blueberry_rails/generators/app_generator.rb

```diff
@@ -101,6 +101,8 @@ def remove_routes_comment_lines
     end
 
     def setup_gems
+      say 'Setting up SimpleForm'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.